test/unit/number-formatting/format-spec.js

```diff
@@ -75,7 +75,7 @@ describe('NumberFormatting format', () => {
             expect(shortFormat(1050000)).toEqual('1.05m');
             expect(shortFormat(10500000)).toEqual('10.5m');
             expect(shortFormat(105000000)).toEqual('105m');
-            expect(shortFormat(1050000000)).toEqual('1,050m');
+            expect(shortFormat(1050000000)).toEqual('1.05bn');

```

Comment:
I think you should add more than a single assertion. E.g. the boundaries on both sides you expect. a larger number. etc.